ProGetAutomation/Functions/Add-ProGetAsset.ps1

```diff
@@ -0,0 +1,59 @@
+function Add-ProGetAsset
+{
+    <#
+        .SYNOPSIS
+        Used to add assets to the proget asset manager. 
+
+        .DESCRIPTION
+        This function adds assets to the proget asset manager. A session, asset name, assetDirectory and filePath is required. 
+        The assetName parameter is the name you wish the asset to be named in proget. 
+        The assetDirectory parameter is the directory you wish the asset to be located in.
+        The FilePath parameter is the path to the file located on your machine. 
+
+        .EXAMPLE
+        # example of adding an asset to proGet Asset Manager if versions is not created it will create the directory.
+        Add-ProGetAsset -Session $session -AssetName 'exampleAsset' -AssetDirectory 'versions' -FilePath 'path/to/file.txt'
+
+        .EXAMPLE
+        # example of adding an asset to proGet Asset Manager if versions or subfolder are not created it will create both directories.

```

Comment:
No comments in examples. The first line of an example should be the code, followed by an empty line, followed by a description of the example. Please update all your examples to follow this format.

Reply:
Done